README.md

```diff
@@ -1,6 +1,9 @@
 # Igbo Dictionary API
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/ijemmao/igbo_api/Igbo%20API%20CI)
```

Comment:
The name for this workflow changed to `integration` so once you update the name here, I think you should be good to go!